bin/improver-nbhood

```diff
@@ -72,8 +75,8 @@ def main():
         radius_or_radii_in_km = args.radii_in_km_by_lead_time[0].split(",")
         lead_times = args.radii_in_km_by_lead_time[1].split(",")
     result = (
-        BasicNeighbourhoodProcessing(
-            radius_or_radii_in_km,
+        NeighbourhoodProcessing(
+            args.kernel_method, radius_or_radii_in_km,
```

Comment:
Well spotted. This was a flaw in previous implementations of the CLI. We'd like to check with @benfitzpatrick before making this change, as we're not sure whether we should also be adding an acceptance test as well. If we need to edit the CLI and add an acceptance test, then this might be better handled as technical debt, otherwise, if the change is just to the CLI, then we could handle this here.